packages/twenty-front/src/modules/settings/data-model/fields/forms/hooks/useFieldMetadataForm.ts

```diff
@@ -41,12 +41,14 @@ const fieldSchema = z.object({
   icon: z.string().startsWith('Icon'),
   label: z.string().min(1),
   defaultValue: z.any(),
-  type: z.enum(
-    Object.values(FieldMetadataType) as [
-      FieldMetadataType,
-      ...FieldMetadataType[],
-    ],
-  ),
+  type: z
```

Comment:
@martmull how is making the type optional fixing the issue?

Reply:
Type is not necessary for relation fields. So I removed it and it frees the save button because before, without type, the data where declared invalid by z

Reply:
Ok, would be great to introduce a relationTargetFieldSchema that would not have the type and a fieldSchema that would have it. (we can build relationTargetFieldSchema on top of fieldSchema maybe)
Otherwise, I feel this is too flexible and will cause bugs